Assets/Scripts/SimplePhysics.cs

```diff
@@ -130,11 +133,10 @@ void Update()
             mVelY += mGravity;
         }
 
-        // TODO: Bug where character accelerates much more quickly in x dir
         private float UseAccel(float accel, float curSpeed, float maxSpeed)
         {
             float newSpeed = accel + curSpeed;
-            return (newSpeed < maxSpeed) ? newSpeed : maxSpeed;
+            return (Mathf.Abs(newSpeed) < maxSpeed) ? newSpeed : maxSpeed;

```

Comment:
It just seems like theoretically it SHOULD be an issue. if you hold left xInput would be negative, meaning maxSpeed would be negative and that condition would always return max speed. 

If you can verify things are working then I'll go ahead and approve but I'm probably going to do a quick pull-and-test myself and then I'll file a bug if I see any issue